test/integration/spec/rtcpeerconnection.js

```diff
@@ -36,19 +36,17 @@ describe('RTCPeerConnection', function() {
     signalingStates.forEach(testAddIceCandidate);
   });
 
-  describe('#getLocalStreams, called from signaling state', () => {
-    signalingStates.forEach(testGetLocalStreams);
-  });
-
-  describe('#getRemoteStreams, called from signaling state', () => {
-    signalingStates.forEach(testGetRemoteStreams);
+  (isChrome ? describe : describe.skip)('#getSenders', () => {
```

Comment:
Why are we skipping for Chrome?

Reply:
@markandrus Actually we are running this test only for Chrome and skipping for FF/Safari as they have native support.